### PR TITLE
[DOC-10237] IdP-initiated SAML flow enabled by default

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-10237] IdP-initiated SAML flow enabled by default

### CODE REVIEW

File: src/current/_data/cloud_releases.csv [hunk 1]
```diff
@@ -87,3 +87,4 @@ date
 2024-05-12
 2024-05-20
 2024-06-12
+2024-06-196
```
Comment (mdlinville) [inline src/current/_data/cloud_releases.csv:90]:
```suggestion
2024-06-19
```

**Review by jaiayu — APPROVED** [review]
Looks good to me, thanks!

File: src/current/cockroachcloud/cloud-org-sso.md [hunk 2]
```diff
@@ -110,10 +110,10 @@ Yes. When Cloud Organization SSO is enabled for your CockroachDB {{ site.data.pr
 
 #### Which SAML-based authentication flows are supported with Cloud Organization SSO?
 
-The following flows are supported:
+After SAML is configured, your users can sign in to the CockroachDB {{ site.data.products.cloud }} Console two different ways:
```
Comment (florence-crl) [inline src/current/cockroachcloud/cloud-org-sso.md:113]: nit

```suggestion
After SAML is configured, your users can sign in to the CockroachDB {{ site.data.products.cloud }} Console in two different ways:
```

File: src/current/cockroachcloud/configure-cloud-org-sso.md [hunk 3]
```diff
@@ -206,6 +206,11 @@ To configure a custom SAML authentication method using the service provider-init
     In Okta, the SAML assertion does not include the `email` field by default, and it must be added. For detailed instructions, refer to [How to Send Attributes via the SAML Assertion](https://support.okta.com/help/s/article/Skipping-assertion-attributes-because-of-schema-mismatch) in the Okta documentation.
 1. (Optional) To configure SCIM provisioning, refer to [Configure SCIM autoprovisioning]({% link cockroachcloud/configure-scim-provisioning.md %}).
 
+After SAML is configured, your users can sign in to the CockroachDB {{ site.data.products.cloud }} Console two different ways:
```
Comment (florence-crl) [inline src/current/cockroachcloud/configure-cloud-org-sso.md:209]: nit

```suggestion
After SAML is configured, your users can sign in to the CockroachDB {{ site.data.products.cloud }} Console in two different ways:
```

**Review by florence-crl — APPROVED** [review]
LGTM pending suggestions (nits)

**Pull request merged**